### DISCUSSION TITLE
M5Stack relay M5-U131 or solid state relay M5-U122

### DISCUSSION THREAD

**Discussion by philipgo** [raomin/ESPAltherma#433] [General]
Hello,

has anyone tried using one of these relays? Will they work for thermostat control?
![grafik](https://github.com/raomin/ESPAltherma/assets/29184757/50a30bb3-486c-4398-83a4-a95c3151ff39)
or
![grafik](https://github.com/raomin/ESPAltherma/assets/29184757/566e75fd-589d-4717-958b-f4083052fc7c)

Thank you!

**Comment by my-SmarthomeGmx** [comment]
I am using the first (dual relay) one for the Smart Grid Control. See here
https://github.com/raomin/ESPAltherma/issues/17#issuecomment-2062148666
It works without issues.

**Reply by philipgo** [reply]
Thank you! I somehow missed your post in the other thread. 

What is the diameter of the wires you use? They look pretty small on thd photo. Is there any signicant current flowing through these wires?

**Reply by my-SmarthomeGmx** [reply]
They are 0,8mm2  
According Daikin Installation Manual for my Altherma they should be 0.5 mm2

![image](https://github.com/raomin/ESPAltherma/assets/135668641/339fdb90-d950-40c4-bccc-a8e9db8bd580)
